dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -78,9 +85,15 @@ final class QueuedChannel implements Channel {
     // avoid creating unnecessary data.
     private volatile boolean shouldRecordQueueMetrics;
 
-    QueuedChannel(LimitedChannel delegate, String channelName, QueuedChannelInstrumentation metrics, int maxQueueSize) {
+    QueuedChannel(
+            LimitedChannel delegate,
+            @Safe String channelName,
+            @Safe String queueType,
```

Comment:
Is queueType basically channelType ? are queue and channel being used interchangeably here ?

Reply:
nope -- the diagram in the readme describes two types of queues (it doesn't go into much detail about sticky, but you can ignore that for most use-cases). There are two queues each request progresses through before it's sent to a remote server.